styleguide/_themes/global/scss/vendor_prefixes.scss

```diff
@@ -52,3 +58,13 @@
   -webkit-filter: sepia(.9);
   filter: sepia(90%);
 }
+
+@mixin text-stroke($width, $color) {
+  -webkit-text-stroke: $width $color;
+}
+
+@mixin fade($duration, $type) {
```

Comment:
While I definitely like having a generic mixin here to specify a duration and easing function, I'm wondering if it makes sense to just set the values you mentioned separately as a default `$duration` and `$type` value for this mixin.  I think this can help avoid inconsistencies in the future, and will allow us to just do `@include fade;` wherever we need this effect.